src/Module/Upgrade/Upgrade1_8_2.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gett\MyparcelBE\Module\Upgrade;
+
+use Gett\MyparcelBE\Constant;
+use Gett\MyparcelBE\Database\Table;
+use MyParcelNL\Sdk\src\Support\Collection;
+
+class Upgrade1_8_2 extends AbstractUpgrade
+{
+    private $carrierConfigurationTable;
+
+    /**
+     * @throws \PrestaShopDatabaseException
+     */
+    public function upgrade(): void
+    {
+        $this->addInsuranceOptionsBeForCarriers();
+    }
+
+    /**
+     * Adds insurance_from_price and insurance_max_amount to carrier configuration if not already present
+     *
+     * @throws \PrestaShopDatabaseException
+     */
+    private function addInsuranceOptionsBeForCarriers(): void
```

Comment:
Is het misschien goed om hier onderscheid te maken naar platform (NL of BE), en om alleen postNL in platform NL eruit te pikken? Voor de rest gelden deze insurance opties toch niet?

Reply:
Ik heb hem gewoon alleen even laten returnen als de plugin Belgisch is, want volgens mij is het heel lastig/intensief om te achterhalen welke carrier bij een prestashop carrier id hoort.